lib/archethic_web/reference_transaction.ex

```diff
@@ -0,0 +1,71 @@
+defmodule ArchethicWeb.ReferenceTransaction do
```

Comment:
To remove confusion about the naming of this module, we could specify that is about web hosting. Otherwise it means a global reference transaction - which is not the case. It's only for AEWeb. 

Maybe `ArchethicWeb.WebHostingController. ReferenceTransaction`